main.c

```diff
@@ -974,14 +1010,26 @@ int main(int argc, char *argv[]) {
 	struct wl_registry *registry = wl_display_get_registry(state.display);
 	wl_registry_add_listener(registry, &registry_listener, NULL);
 	wl_display_roundtrip(state.display);
+	wl_display_roundtrip(state.display);
```

Comment:
Sorry for the necro here but I was studying the code and wondered whether this line got doubled by accident.

Reply:
No, two roundtrips are necessary: one to fetch globals, another to fetch the seat name.